command/namespace_lookup.go

```diff
@@ -21,7 +21,7 @@ func (c *NamespaceLookupCommand) Synopsis() string {
 
 func (c *NamespaceLookupCommand) Help() string {
 	helpText := `
-Usage: vault namespace create [options] PATH
+Usage: vault namespace lookup [options] PATH
 
   Create a child namespace. The namespace created will be relative to the
```

Comment:
I think this description should be updated to refer to `lookup` as well, correct?

Reply:
Thanks, good catch, I addressed this.